packages/sdk-v2/src/sdk/goodid/react.ts

```diff
@@ -147,3 +145,80 @@ export const useAggregatedCertificates = (account: string): AggregatedCertificat
     });
   }, [certificates]);
 };
+
+// could be the configuration has to be set on the native side (on root-level)
+// at least the permission android.permission.ACCESS_FINE_LOCATION has to be added
+// in order for below hook to work
+
+// GeoLocation.setRNConfiguration({
+//   skipPermissionRequests: false,
+//   authorizationLevel: "whenInUse",
+//   locationProvider: "auto"
+// });
+
+export interface LocationState {
+  location: [number, number] | null;
+  error?: string | null;
+}
+
+export const useLocation = () => {
```

Comment:
useGeoLocation, otherwise may be confused with navigation's location

Reply:
done here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/136/commits/147356f77fe0049b9a7e3ef71c0b1280933089ca